src/backend/distributed/commands/call.c

```diff
@@ -46,6 +47,9 @@
 static bool CallFuncExprRemotely(CallStmt *callStmt,
 								 DistObjectCacheEntry *procedure,
 								 FuncExpr *funcExpr, DestReceiver *dest);
+#if PG_VERSION_NUM >= PG_VERSION_14
```

Comment:
You should explicitly include "distributed/pg_version_constants.h" header after including "postgres.h":

```
#include "postgres.h"
#include "funcapi.h"

#include "distributed/pg_version_constants.h"


```

Otherwise PG_VERSION_14 might take a random value and it might be that this is executed in pg12 and pg13 as well. We might not be getting an error because `PROARGMODE_OUT` exists in 13.1 as far as I can see and it probably exists in pg12 as well.

Reply:
Hmm, I didn't know this.